### PR TITLE
Changes according to MDVA-10804

### PR DESCRIPTION
**Pull request by isitnikov**
Documentation must be updated, because customer found other case, when our Upgrading guide provides incorrect steps.

## This PR is a:
[ ] New topic
[*] Content fix or rewrite
[ ] Bug fix or improvement
 

## Summary
 
When this pull request is merged, it will provide additional information how to upgrade Magento when it's used as a dependency.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=1964) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=1964) before we can accept your contribution.<br/>

**Comment by yyevgenii** [comment]
@isitnikov you have to make the same changes in the `guides/v2.0/comp-mgr/cli/cli-upgrade.md` and `guides/v2.1/comp-mgr/cli/cli-upgrade.md`

**Comment by jeff-matthews** [comment]
@dshevtsov, assigning to you since you're also working on general upgrade instructions for 2.3 (MAGEDOC-2812).

**Comment by dshevtsov** [comment]
Created an internal issue: MAGEDOC-2984

File: guides/v2.2/comp-mgr/cli/cli-upgrade.md [hunk 1]
```diff
@@ -71,17 +71,45 @@ To prevent access to your store while it's being upgraded, put your store in mai
...
 		composer update
 
-	To upgrade to Magento EE version 2.0.11, enter:
+	To upgrade to Magento EE version 2.2.2, enter:
 
-		composer require magento/product-enterprise-edition 2.0.11 --no-update
+		composer require magento/product-enterprise-edition 2.2.2 --no-update
+		composer update
+		
+	If you have own *composer.json* where Magento package specified as a dependency. For example:
```
Comment (buskamuza) [inline guides/v2.2/comp-mgr/cli/cli-upgrade.md]: What's the use case for this?
If they want to depend on Magento, they should depend on "product", not "project". Project should be specifically used for a **project**, not as a dependency.

Comment (isitnikov) [reply guides/v2.2/comp-mgr/cli/cli-upgrade.md]: @buskamuza is it described in our official documentation? Just out client made dependency from the *magento/project-enterprise-edition* and I updated this guide especially for such cases.

**Comment by dshevtsov** [comment]
@isitnikov this case is a workaround and is not typical for general upgrading process.

**Pull request closed without merging**